metis-preview/metis-preview-service/src/main/java/eu/europeana/metis/preview/service/ZipService.java

```diff
@@ -25,32 +24,31 @@
 
   private static final Logger LOGGER = LoggerFactory.getLogger(ZipService.class);
 
-  public List<String> readFileToStringList(InputStream inputStream) throws ZipFileException {
+  public List<String> readFileToStringList(MultipartFile file) throws ZipFileException {
 
-    //TODO use a different zip library based on streams so we don't have to create files
-    //and delete them afterward
-    List<String> xmls = new ArrayList<>();
+    List<String> records = new ArrayList<>();
+    
     try {
-      String prefix = String.valueOf(new Date().getTime());
```

Comment:
Why was this part modified? I understand that it was probably copied from validation module but here it was actually cleaner by using the default implementation of java for temp files.
Also the logging statements were removed. Please revert the handling of temp files to the previous version.

Reply:
The files are simply created in another manner now, so the prefix is not required. The missing logging statements have been fixed.

Reply:
Why do you think this is a better approach? There is a default implementation from java, we don't have to reinvent it if possible. Also the methods from java io have the benefit of avoiding same file name collisions.

Reply:
I agree with what you're saying. I've modified the implementation to work with the standard Java IO classes again. We should consider changing this in the validation service too.

Reply:
Yes indeed we  should modify that part too on another ticket though.